discovery-infra/test_infra/controllers/node_controllers/node.py

```diff
@@ -145,3 +145,12 @@ def attach_test_disk(self, disk_size, bootable=False):
 
     def detach_all_test_disks(self):
         self.node_controller.detach_all_test_disks(self.name)
+
+    def attach_interface(self, node_name, network_xml, target_interface="vnet3"):
+        return self.node_controller.attach_interface(node_name, network_xml, target_interface)
+
+    def destroy_network(self, network):
+        self.node_controller.destroy_network(network)
+
+    def undefine_interface(self, node_name, mac):
```

Comment:
why not `self.node` instead of passing `node_name` to the method?

Reply:
fixed